app/static/js/createEvents.js

```diff
@@ -31,13 +31,16 @@ $(document).ready(function() {
     }
   });
 
-  if (navigator.userAgent.indexOf("Firefox") != -1 || navigator.userAgent.indexOf("Safari") != -1) {
+  // everything except Chrome
+  if (navigator.userAgent.indexOf("Chrome") == -1) {
+    console.log ("But we have Chrome")
+    console.log(navigator.userAgent)
```

Comment:
Remove test console.log statements

Reply:
We need print statements and console log statements for monday. They will be removed once issue is resolved.